src/app/components/AccountHistory.tsx

```diff
@@ -55,11 +56,11 @@ function OrdersTabs() {
   const openOrders = useAppSelector(selectOpenOrders);
 
   function tabClass(isActive: boolean) {
-    return (
-      "tab w-max no-underline h-full py-3 tab-border-1 font-bold text-sm uppercase leading-4" +
-      (isActive
-        ? " tab-active tab-bordered text-accent-focus !border-accent"
-        : "")
+    return twMerge(
+      "px-4 text-secondary-content cursor-pointer w-max no-underline h-full py-3 font-bold text-sm uppercase leading-4",
+      isActive
+        ? "tab-active text-[#CBFC42] border-b-2 border-[#CBFC42]"
+        : "hover:text-base-content"
```

Comment:
Looks good to me, but I am wondering: how did you find places in the code that needed twMerge utility? How can we ensure/check we don't miss any problematic code?

Reply:
I followed the updating guide and tried to identify all instances of obsolete classes, such as those related to -focus or tabs. I updated the color system from HSL (where it was used in the code) to OKLCH and adapted the code to align with the previous design.

The main changes were applied to the `tabClass` function. This is the only place where I used `twMerge`, as I could not get `hover:text-base-content` to take precedence over `text-secondary-content`otherwise.

I compared the current design with the "new" one as thoroughly as possible and couldn’t identify any differences. However, to ensure accuracy, it would be beneficial to have multiple people review it.